### PR TITLE
Add the example for plotting GPS velocities

### PR DESCRIPTION
**Pull request by seisman**
Preview: https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/ex015/examples/ex015/

Closes https://github.com/gmt-china/GMT_docs/issues/845.

### CODE REVIEW

**Review by wangliang1989 — APPROVED** [review]

File: source/examples/ex015/index.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+绘制 GPS 速度场
+===============
+
+:示例贡献者: 李黎明
+
+----
+
+本示例展示如何使用 :doc:`/module/velo` 模块绘制 GPS 速度场。
+
+示例中所使用的 GPS 数据来自于 `Zhao et al., Geod. Geodyn., 2015 <https://doi.org/10.1016/j.geog.2014.12.006>`__
```
Comment (core-man) [inline source/examples/ex015/index.rst:12]:
```suggestion
示例中所使用的 GPS 数据来自于 `Zhao et al. (2015) <https://doi.org/10.1016/j.geog.2014.12.006>`__
```

File: source/examples/ex015/index.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+的补充材料并根据 :doc:`/module/velo` 模块的要求做简单修改。
```
Comment (core-man) [inline source/examples/ex015/index.rst:13]:
```suggestion
的补充材料，根据 :doc:`/module/velo` 模块的要求做简单修改。
```

File: source/examples/ex015/ex015.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# 绘制国界
+gmt plot CN-border-L1.gmt -W1p,-
+
+# 绘制GPS速度场
+# -Se后面两个参数表示：速度值为1的矢量的长度/置信度0.95
+#    +f 设置文本的大小，0 表示不显示标注
+# -W控制矢量线及误差椭圆的轮廓的宽度,颜色,线型
+# -G矢量头的填充色
+# -A控制矢量的属性，0.15c是矢量头的大小，+e表示在矢量尾端绘制箭头，+p0.75p,blue设置矢量箭头轮廓的属性
```
Comment (core-man) [inline source/examples/ex015/ex015.sh:22]:
```suggestion
# 绘制 GPS 速度场
# -Se 后面两个参数表示：
#    速度值为 1 的矢量的长度（0.05c）/置信度（0.95）
#    +f 设置文本的大小，0 表示不显示标注
# -A 控制矢量的属性，0.15c 是矢量头的大小，+e 表示在矢量尾端绘制箭头，+p0.75p,blue 设置矢量箭头轮廓的属性
# -W 控制矢量线及误差椭圆的轮廓的宽度,颜色,线型
# -G 矢量头的填充色
```

File: source/examples/ex015/ex015.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+gmt velo gps_campagin.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,blue -Gblue -W0.5p,blue
+gmt velo gps_continuous.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -Gred -W0.5p,red
```
Comment (core-man) [inline source/examples/ex015/ex015.sh:24]: Same as the following commands?
```suggestion
gmt velo gps_campagin.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,blue -W0.5p,blue -Gblue
gmt velo gps_continuous.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -W0.5p,red -Gred
```

File: source/examples/ex015/ex015.sh [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+echo 0.05 0.25 20 0 1 1 0 | gmt velo -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -Gred -W0.5p,red
+gmt text -F+f6p+jML -M <<- EOF
+> 0.4 0.50 0.25 2c c
+Continuous Station
+1998-2014
+> 0.4 0.25 0.25 2c c
+Campaign Stations
+2009-2014
+EOF
+gmt inset end
```
Comment (core-man) [inline source/examples/ex015/ex015.sh:41]:
```suggestion

gmt inset end
```

File: source/examples/ex015/ex015.sh [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+gmt velo gps_continuous.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -Gred -W0.5p,red
+
+# 添加图例
+# 此处在底图右下角，设置了一个宽 3.5c 高 2c 的底图
+# 底图范围为 0/1/0/1，矢量箭头和文字的位置需对应该小图中的位置
+gmt inset begin -DjBR -F+gwhite -R0/1/0/1 -JX3.5c/2c
+echo 0.05 0.75 '20\2611 mm/yr' | gmt text -F+f7p+jML
+echo 0.05 0.50 20 0 1 1 0 | gmt velo -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,blue -Gblue -W0.5p,blue
+echo 0.05 0.25 20 0 1 1 0 | gmt velo -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -Gred -W0.5p,red
+gmt text -F+f6p+jML -M <<- EOF
```
Comment (core-man) [inline source/examples/ex015/ex015.sh:33]: Do we need the `-`?
```suggestion
gmt text -F+f6p+jML -M << EOF
```

File: source/examples/ex015/ex015.sh [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+# -W控制矢量线及误差椭圆的轮廓的宽度,颜色,线型
+# -G矢量头的填充色
+# -A控制矢量的属性，0.15c是矢量头的大小，+e表示在矢量尾端绘制箭头，+p0.75p,blue设置矢量箭头轮廓的属性
+gmt velo gps_campagin.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,blue -Gblue -W0.5p,blue
+gmt velo gps_continuous.txt -Se0.05c/0.95+f0 -A0.15c+e+p0.5p,red -Gred -W0.5p,red
+
+# 添加图例
+# 此处在底图右下角，设置了一个宽 3.5c 高 2c 的底图
+# 底图范围为 0/1/0/1，矢量箭头和文字的位置需对应该小图中的位置
+gmt inset begin -DjBR -F+gwhite -R0/1/0/1 -JX3.5c/2c
```
Comment (core-man) [inline source/examples/ex015/ex015.sh:29]:
```suggestion
gmt inset begin -R0/1/0/1 -JX3.5c/2c -DjBR -F+gwhite
```

**Review by core-man — APPROVED** [review]

**Pull request merged**